docs/developer-center/guides/04-Login-to-CARTO-Platform.md

```diff
@@ -1 +1,76 @@
-## Login to CARTOplatform
\ No newline at end of file
+## Login to CARTO Platform
+
+In this guide, our purpose is to focus on the basics of authentication in CARTOframes. There's a full [Authorization Fundamentals](https://carto.com/developers/fundamentals/authorization/) at CARTO guide to understand how to configure and manage API Keys.
+
+> To use CARTOframes is not *always* needed to be authenticated.
+
+It is needed to set up the user credentials to use Data Services, Data Enrichment or the Data Observatory, between others. In these cases, it's required to have a CARTO account. Once the user has created an account, the credentials can be found at **http://johnsmith.carto.com/your_apps.** for user `johnsmith`, and it should be a **Master** API Key:
```

Comment:
Maybe a link to the signup page when you say it's required to have a CARTO account?